ddp-workspace/projects/ddp-pancan/src/app/components/welcome/welcome.component.html

```diff
@@ -1 +1,112 @@
-<main class="main">welcome!</main>
+<main class="main">
+    <section class="introduction">
+        <h2 class="title" translate>App.HomePage.IntroductionSection.Title</h2>
+        <div class="text" translate>App.HomePage.IntroductionSection.Text</div>
```

Comment:
let's use p instead of div here for a11y

Reply:
Fixed at 5934f4245f5e114a6153ffdb7e957e24f6eb646a